### PR TITLE
fix(core): drop some excess dispatches of `Input`-event

### LINKED ISSUE #879: 🐞 - Input element masked with date range mask fires InputEvent twice first time
### Which package(s) are the source of the bug?

@maskito/core, @maskito/kit

### Playground Link

https://stackblitz.com/edit/xd2ihb?file=index.ts&hideExplorer=1

### Description

When using date range mask and typing first symbol in input - it emits `InputEvent` twice, check out stackblitz, type for ex. `1` in input and check console logs. I guess assigning placeholder firstly and actual `1D/MM/YYYY - DD/MM/YYYY` after causing this issue. Can maskito handle this?

### Maskito version

1.9.0

### Which browsers have you used?

- [X] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### Which operating systems have you used?

- [X] macOS
- [ ] Windows
- [ ] Linux
- [ ] iOS
- [ ] Android

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Type

What kind of change does this PR introduce?

- [X] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

Closes #879

### CODE REVIEW

File: projects/demo-integrations/src/tests/component-testing/placeholder/placeholder-dispatch-input-events.cy.ts
Comment (nsbarsukov) [inline projects/demo-integrations/src/tests/component-testing/placeholder/placeholder-dispatch-input-events.cy.ts:1]: Before all changes:
before

Comment (nsbarsukov) [reply projects/demo-integrations/src/tests/component-testing/placeholder/placeholder-dispatch-input-events.cy.ts:1]: After all changes:

after

File: projects/kit/src/lib/processors/with-placeholder.ts [hunk 1]
```diff
@@ -63,8 +63,8 @@ export function maskitoWithPlaceholder(
             },
         ],
         postprocessors: [
-            ({value, selection}, initialElementState) =>
-                initialElementState.value && (focused || !focusedOnly)
```
Comment (nsbarsukov) [inline projects/kit/src/lib/processors/with-placeholder.ts:67]: It is legacy code part that causes excess `Input`-dispatching.

It is not more required after closing of this issue:
* https://github.com/taiga-family/maskito/pull/778

**Comment by nsbarsukov** [comment]
This bugfix will be released in Maskito 2.0.0 (the expected release date is the second half of January).

**Review by splincode — APPROVED** [review]

**Pull request merged**